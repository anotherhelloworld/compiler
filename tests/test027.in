var
  x: 1..10;
  y: 'a'..'z';
  z: pear..orange;