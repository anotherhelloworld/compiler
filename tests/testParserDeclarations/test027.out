Procedure   foo   0
   begin
      for
            i
         :=
            0
      true
         10
      do
         begin
         end
   end

